packages/cfd/src/Components/cfd-personal-details-form.tsx

```diff
@@ -428,19 +412,19 @@ const CFDPersonalDetailsForm = ({
                                                 <InputField
                                                     id='real_mt5_tax_identification_number'
                                                     name='tax_identification_number'
-                                                    label={tin_field_label}
-                                                    placeholder={tin_field_label}
+                                                    label={localize('Tax identification number*')}
+                                                    placeholder={localize('Tax identification number*')}
```

Comment:
Can we use `<Localize />` instead?

Reply:
I would say if we go this way the whole form need to be refactored, which we can take as a separate card maybe

Reply:
okiee 👌